Source/Fuse.iOS.TextRenderer/TextRenderer.uno

```diff
@@ -47,7 +48,9 @@ namespace Fuse.iOS.Bindings
 
 			_font = Fuse.Controls.Native.iOS.FontCache.Get(control.Font.Descriptors[0], control.FontSize * control.Viewport.PixelsPerPoint);
 
-			_textColor = ToUIColor(control.TextColor);
+			//iOS Text rendering fails to apply opacity to emoji, therefore we apply opacity on our own
+			_textColor = ToUIColor(float4(control.TextColor.XYZ,1));
+			TextOpacity = control.TextColor.W;
```

Comment:
I don't think this is correct. We've previously defined TextColor to only apply to non-colored glyphs (typically emoji, but can also apply to colorfonts in the future). That is, it can be thought of as a "default text color unless the font specifies otherwise", not as a multiplier on the font-data.

If the user wants to fade out all text, `Opacity` should be used instead.

Reply:
I don't see that we actively defined what should happen before, there's no notes on it and it behaves differently per-platform. On Android the colored glyphs fade with the provided text opacity.  Given Android is the dominant platform, and the behaviour seems reasonable, I think it's the correct definition.

I can't see any use-case why the user would want the rest of the text faded, but not the colored glyphs -- it leads to a jarring visual contrast.

If we instead adopt iOS's approach, we're left with Android, thus most devices, not rendering consistently.

Reply:
HTML/CSS also fades out the emoji opacity (tested on Firefox).

Reply:
This has been defined, but probably not documented anywhere.

However, if HTML/CSS does it, I think that's a good argument for us to do it as well. But please fix these things as separate commits.

Reply:
I'm talking separate commits, not separate PRs. It's just easier to read what happened if there's one concern at the time.